deepchem/models/torch_models/layers.py

```diff
@@ -2972,3 +2972,83 @@ def _dynamic_partition(self, input_tensor: torch.Tensor,
         partitions = [input_tensor[mask] for mask in partition_masks]
 
         return partitions
+
+
+class DTNNEmbedding(nn.Module):
+    """DTNNEmbedding layer for DTNN model.
+    Assign initial atomic descriptors. [1]_
```

Comment:
Let's add a longer description here. Say about a small paragraph that explains what this layer does conceptually.

Reply:
Done